lib/scsi/scsi_bdev.c

```diff
@@ -769,10 +772,10 @@ spdk_bdev_scsi_inquiry(struct spdk_bdev *bdev, struct spdk_scsi_task *task,
 	return hlen + len;
 
 inq_error:
-	task->data_transferred = 0;
```

Comment:
This change (removing data_transferred = 0) breaks the scsi_bdev_ut inquiry overflow test.  Is the removal intentional?